src/main/webapp/js/httputil.js

```diff
@@ -0,0 +1,34 @@
+function HttpUtil() {
```

Comment:
The README on https://github.com/TransbankDevelopers/transbank-sdk-js-onepay/ says that HttpUtil is included on the SDK. Why do you need to add this code then?

Reply:
In the webpack process made by @eugenioLopezRamos thi function did not leave as public and we have talked about this is not an SDK functionality and who integrate it could chose the better way to call the "callback". In the new js that example are using this function does not exist.

Reply:
Sounds good. But then you need to fix the README on the JS SDK to stop using `HttpUtil` as the instructions won't work anymore.

Reply:
I will include this change in the next PR with the browser support